core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/MultiCloudWorkspaceService.scala

```diff
@@ -635,6 +625,7 @@ class MultiCloudWorkspaceService(override val ctx: RawlsRequestContext,
                                                                   workspaceRequest.name,
                                                                   spendProfileId,
```

Comment:
Don't we also want to start Leo when creating a `protectedWorkspaceWithSpendProfile` (line 616)?

This is the sort of reason why I don't like having the 2 different method, vs. just making whether or not it is protected a boolean arg on a single method.

And do we need to do anything on clone?

Reply:
Yeah good call, I refactored a bit in this [commit](https://github.com/broadinstitute/rawls/pull/2552/commits/ca812746a7305aca5f73cb573f007ce8d6739c2e) to coalesce the 2 methods into a single one consuming an optional policy inputs object. 

The enable/disable was in the cloneworkspace method of this class. WSM now does the right thing on clone and pulls the app IDs from the source workspace and enables the apps properly. I tested in a BEE and got a WDS booted up on a clone with this change.